styles.css

```diff
@@ -343,3 +346,276 @@ footer {
+  }
+
+  .grid2 {
+    grid-row: 3/4;
+    grid-column: 1/8;
+    flex-direction: row;
+  }
+
+  .grid3 > h1 { display: none; }
+  .grid3 > p { display: none; }
+
+  .aboutMe > h1 { line-height: 81.5px; }
+
+  .aboutMe > p {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .grid2 > div > h1 {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .grid2 > div > p {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .grid3 {
+    grid-row: 5/6;
+    grid-column: 1/2;
+    background-image: url("img/deskGrid3.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    justify-content: end;
+  }
+
+  .grid3 > ul { display: none; }
+  .grid3 > button { width: 25.6vw; }
+
+  .project2 { gap: 0; }
+
+  .grid4 {
+    grid-row: 5/6;
+    grid-column: 3/4;
+    background-image: url("img/deskGrid4.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 156px;
+  }
+  .grid4 > button { display: none; }
+
+  .grid5 {
+    grid-row: 5/6;
+    grid-column: 5/6;
+    background-image: url("img/deskGrid5.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 200px;
+  }
+  .grid5 > button { display: none; }
+
+  .grid6 {
+    grid-row: 7/8;
+    grid-column: 1/2;
+    gap: 24px;
+    padding-top: 156px;
+    background-image: url("img/deskGrid6.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  .grid6 > button { display: none; }
+
+  .grid7 {
+    grid-row: 7/8;
+    grid-column: 3/4;
+    background-image: url("img/deskGrid4.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 156px;
+  }
+  .grid7 > button { display: none; }
+
+  .grid8 {
+    grid-row: 7/8;
+    grid-column: 5/6;
+    background-image: url("img/deskGrid5.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    gap: 24px;
+    padding-top: 200px;
+  }
+  .grid8 > button { display: none; }
+
+  #about {
+    background-image: url("img/deskAbout.png");
+    background-repeat: no-repeat;
+    background-position: 100% 8.3%;
+  }
+
+  .aboutMe { width: 39.3vw; }
+
+  .skills {
+    flex-direction: row;
+    gap: 10.5vw;
+    justify-content: center;
+  }
+
+  #contact > div {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .deskNav > ul {
+    list-style: none;
+    display: flex;
+    gap: 2.2vw;
+    align-items: center;
+    padding-top: 5px;
+  }
+
+  footer > .socialIcons > ul {
+    flex-direction: row;
+    padding-top: 15px;
+    padding-bottom: 16px;
+  }
+
+  .deskNav {
+    display: flex;
+  }
+
+  .deskNav > ul > li {
+    display: inline;
+  }
+
+  .navigationContent > a {
+    padding: 10px 0.8vw;
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .deskNav > ul > li > a {
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    color: #fff;
+    font-weight: 600;
+    line-height: 20px;
+  }
+
```

Comment:
- [ ] Kindly add more space between the social icons on the footer to fit the FIGMA design

![image](https://user-images.githubusercontent.com/93630700/206734497-09d940ff-6376-43de-b9a8-d0b2c4bcbc5f.png)